transaction/_transaction.py

```diff
@@ -504,19 +508,19 @@ def note(self, text):
         """
         text = text.strip()
         if self.description:
-            self.description += "\n" + text
+            self.description += u"\n" + text
         else:
             self.description = text
 
     def setUser(self, user_name, path="/"):
         """ See ITransaction.
         """
-        self.user = "%s %s" % (path, user_name)
+        self.user = u"%s %s" % (path, user_name)
```

Comment:
If `setUser` is no longer part of the `ITransaction` interface, maybe we should be making `user` (and `description` into properties which do typechecking?